test/conformance/api/v1/volumes_test.go

```diff
@@ -220,15 +211,8 @@ func TestSecretVolume(t *testing.T) {
 		},
 	})
 
-	withOptionalBadVolume := WithVolume("blah", "/does/not/matter", corev1.VolumeSource{
-		Secret: &corev1.SecretVolumeSource{
-			SecretName: "does-not-exist",
-			Optional:   ptr.Bool(true),
```

Comment:
I wasn't aware of this Optional flag. Should we respect it in the code above and not check option secrets?